### PR TITLE
Update UltraPermissions_Dutch.lang

### PR DESCRIPTION
**Pull request by Wepwawetcode**

### CODE REVIEW

File: Translations/UltraPermissions_Dutch.lang [hunk 1]
```diff
@@ -1,172 +1,172 @@
...
-EDITING_SUFFIX_TYPEINNONE: Typ **geen** om geen achtervoegsel te hebben
-CREATE_GROUP_TITLE: Groep maken
-CREATE_GROUP_SUBTITLE: Voer een groepsnaam in
-CREATE_GROUP_NOSPACES: De groepsnaam mag geen spaties bevatten
-CREATE_GROUP_NOCOLORS: Gebruik geen chatkleuren in de groepsnaam
-CREATE_GROUP_NAMETAKEN: Deze groepsnaam is al in gebruik
-RENAME_GROUP_TITLE: Naam van groep wijzigen
-RENAME_GROUP_SUBTITLE: Voer een nieuwe naam in
-CONFIRM_DELETION_OF: Bevestig verwijdering van %1
+REPORT: Aangeven		PERMISSION: Machtiging
```
Comment (rlnt) [inline Translations/UltraPermissions_Dutch.lang]: This line till line 58 has multiple phrases in one line. Also there are several phrases that don't belong to UltraPermissions.

**Review by rlnt — CHANGES_REQUESTED** [review]
Please check the file before submitting it. There is a lot of wrong formatting and wrong phases. Please correct that.

File: Translations/UltraPermissions_Dutch.lang (hunk 1 above)
Comment (Wepwawetcode) [reply Translations/UltraPermissions_Dutch.lang]: not sure hop that happened

**Review by rlnt — APPROVED** [review]
Thanks for contributing!

**Pull request merged**